lang/en.json

```diff
@@ -213,7 +213,8 @@
         "header": "Comparing {products, plural, one {# Product} other {# Products}}",
         "remove": "Remove",
         "no_remove": "At least 2 products are needed to make a valid comparison.",
-        "add_to_cart": "Add to cart"
+        "add_to_cart": "Add to cart",
+        "no_compare": "You must select at least two products to compare"
```

Comment:
I think it would be better to give more comprehensive name for this message, for example "product_quantity_validation", so with the name of section the full path to this message will be "compare.product_quantity_validation". For me it sounds much more understandable than "compare.no_compare"
I can see you choose that name as it's similar to "no_remove", but still :)
It's not a major point though, so decision is up to you